docs/internal-developers/block-client-apis/checkout/checkout-api.md

```diff
@@ -117,3 +117,92 @@ This hook is used to expose all the interfaces for the registered payment method
 _Why don't payment methods just implement this hook_?
 
 The contract is established through props fed to the payment method components via props. This allows us to avoid having to expose the hook publicly and experiment with how the props are retrieved and exposed in the future.
+
+## Examples
+
+### Passing a value from the client through to server side payment processing
+
+In this example, lets pass some data from the BACs payment method to the server. Registration of BACS looks like this:
+
+```js
+const bankTransferPaymentMethod = {
+	name: PAYMENT_METHOD_NAME,
+	label: <Label />,
+	content: <Content />,
+	edit: <Content />,
+	canMakePayment: () => true,
+	ariaLabel: label,
+	supports: {
+		features: settings?.supports ?? [],
+	},
+};
+```
+
+If we look a the `Content` component, we can see it defined as follows:
+
+```js
+const Content = () => {
+	return decodeEntities( settings.description || '' );
+};
+```
+
+Payment method components are passed, by default, everything from the `usePaymentMethodInterface` hook. So we can consume this in our component like so:
```

Comment:
Should we add a link to this doc for `usePaymentMethodInterface` https://github.com/woocommerce/woocommerce-blocks/blob/trunk/docs/internal-developers/block-client-apis/checkout/checkout-api.md#usepaymentmethodinterface

Reply:
Ill add it.